pkgs/development/python-modules/takethetime/default.nix

```diff
@@ -25,13 +25,6 @@ buildPythonPackage {
 
```

Comment:
Please disable test_average and test_duration since they seem to be timing sensitive

Reply:
Those are the only two tests, should we remove `pytestCheckHook` entirely and rely on `pythonImportsCheck`?

Reply:
Sure, if there are no other tests. I don't think there is any advantage to fetch from pypi.

Reply:
Sure, if there are no other tests. I don't think there is any advantage to fetch from pypi.